docs/container-set-template.md

```diff
@@ -116,3 +116,26 @@ Example B: Lopsided requests, e.g. `a -> b` where `a` is cheap and `b` is expens
 Can you see the problem here? `a` only has small requests, but the container set will use the  total of all requests. So it's as if you're using all that GPU for 10h. This will be expensive.
 
 Solution: do not use container set when you have lopsided requests.
+
+## Container Set Retries
+
+> v3.3 and after
+
+Container Set Retry policies describes how to retry a container nodes in the container set if it fails.
+
+Number of retries(default 0) and sleep duration between retries(default 0s, instant retry) can be set.
+
+The container won't retry if it's unable to locate the command.
+
+Here is an example of a Container Set Template with `retryStrategy`:
+
+```yaml
+      containerSet:
+        containers:
+          - name: retry-containerset
+            image: alpine:latest
+            retryStrategy:
+              limit: "3"
+            command: [ sh, -c ]
+            args: [ "echo intentional failure; exit 1" ]
+```
```

Comment:
Modifying this since the example was incorrect:

```suggestion
\```

!!! Note
    A container set will not be retried if a container's `command` cannot be located.
    As it will fail each time, the retry logic is short-circuited.
```
- should specify that the _entire_ container set is not retried if a _single_ container's `command` cannot be located. That should answer the question in the issue

Reply:
Follow-up [comment](https://github.com/argoproj/argo-workflows/pull/12809#discussion_r1539771062) on style issues